EmbeddedStock/global.json

```diff
@@ -0,0 +1,6 @@
+{
+"projects": [ "src", "test" ],
+"sdk": {
+    "version": "1.0.0-preview2-1-003177"
```

Comment:
why? vi bruger ASP.NET core 1.1.0?
Also: https://ievangelist.github.io/blog/the-global-json/
Fra hans tekst: _"I always immediately delete the sdk property. If omitted the tooling simply assumes the latest SDK installed."_

Normalt er det fint nok at have versionsnumre til sine dependencies, så noget ikke går i stykker, fordi det bruger en nyere version, som måske ikke er compatible med koden. Men siden vi bruger 1.1.0 i resten af projektet, så skal denne version vel også afspejle dette?